src/molecules/Menu.js

```diff
@@ -56,7 +56,7 @@ class Menu extends React.Component {
         {menu.map((item, index) => (
           <StyledList key={index}>
             <StyledLink exact to={match.url + item.url}>
-              <Text textBold small>
+              <Text textBold small letterSpacing>
```

Comment:
This is generally not a good practice, as with the `opacity` prop you had some time ago.

The prop itself is okay, but it _should have some value_. So, instead of it always having a fixed value, let the value be parametric:

```jsx
<Text letterSpacing={1.5} />
```

This way your Text component is far more powerful, as it can have whichever letter spacing you want.